v2/src/components/appInfoForm/index.tsx

```diff
@@ -667,11 +671,7 @@ export default class AppInfoForm extends React.PureComponent<PropsWithChildren<P
                     ) {
                         // if the netlify api route checkbox is set to true
                         // the api base path can only start with `/.netlify/functions`
-                        validationErrors.apiBasePath = "apiBasePath should begin with '/.netlify/functions/' when using Netlify Serverless Functions."
-                    } else if (netlifyApiRouteUsed && validApiBasePath === "/.netlify/functions/") {
```

Comment:
@nkshah2 I've removed this condition as the `validApiBasePath` will never be equal to `/.netlify/functions/` as the path normalization function we use removes the trailing `/`